web/src/lib/static/loadQuickActions.ts

```diff
@@ -0,0 +1,22 @@
+import { QuickAction} from "@/lib/types/types";
+import fs from "fs";
+import path from "path";
+import {convertCertificationMd} from "../utils/markdownReader";
+
+
+const quickActionsDir = path.join(process.cwd(), "..", "content", "src", "licenses");
```

Comment:
[pebble] is the path correct? I think it should reference the quick-actions folder?